File: Consultorio.API/Consultorio.API/Model/Paciente.cs
namespace Consultorio.API.Model
{
    public class Paciente : Entity
    {
        public string? Observacao { get; set; }
        public int PessoaID { get; set; }
        public virtual Pessoa Pessoa { get; set; }
        public virtual ICollection<Consulta>? Consultas { get; set; }
    }
}

File: Consultorio.App/src/app/pages/pacientes/models/Paciente.ts
import { Genero } from '../../../enum/Genero.enum';
import { TipoSanguineo } from '../../../enum/TipoSanguineo.enum';
import { Consulta } from '../../consultas/model/consulta';

export interface Paciente {
  id: number;
  pessoaId: number;
  nome: string;
  nomeSocial: string;
  cpf: string;
  dataNascimento: Date;
  email: string;
  tipoSanguineo: TipoSanguineo;
  genero: Genero;
  cep: string;
  bairro: string;
  endereco: string;
  telefone: string;
  observacao: string;
  consultas: Consulta[];
}
